python/whylogs/api/logger/experimental/logger/actor/thread_rolling_logger.py

```diff
@@ -111,11 +116,9 @@ def to_views(self) -> List[DatasetProfileView]:
             result_set = self._target.get_result_set(dataset_timestamp=self._dataset_timestamp)
             segments = result_set.segments() or []
             return [it for it in [result_set.view(segment) for segment in segments] if it is not None]
-        elif isinstance(self._target, DatasetProfile):
+        else:
             return [self._target.view()]
 
-        raise Exception("Unknown profile type")
```

Comment:
Raising a plain `Exception` is probably  not optimal, but do we not want to throw  here?

Reply:
The exception wasn't needed, the type check is exhaustive, even for mypy.